src/Aurora.Infrastructure/Scrapers/FootFetishBooruImageGifScraper.cs

```diff
@@ -17,7 +18,7 @@ namespace Aurora.Infrastructure.Scrapers
 {
     public class FootFetishBooruImageGifScraper : IOptionScraper
     {
-        private const int _itemsPerPage = 20;
+        internal const int _itemsPerPage = 20;
```

Comment:
probably create separate config database table for all out scrappers rather than store such data in ones

Reply:
There is no need to put this into a configurations, because it is a constant within the FootFetishBooru website, so we cannot change it. However, I have moved this constant into a separate class to not use this hack with internal